.github/workflows/rebase-sandbox-with-master.yml

```diff
@@ -0,0 +1,27 @@
+# Github action to rebase sandbox with master
+name: rebase-sandbox-with-master
+
+# Run every Monday at 12:30 UTC (5:30pm Sunday PDT)
+on:
+  schedule:
+    - cron: '30 0 * * Mon'
+
+jobs:
+  rebase-and-push:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out the Sandbox branch from Github
+        uses: actions/checkout@v3
+        with:
+          ref: sandbox
+
+      - name: Update sandbox on Pantheon
```

Comment:
This doesn't technically update anything on Pantheon, does it? It looks like it's just doing everything with the git repo on GitHub. I don't see any problem with it, just that the name is misleading.

Reply:
Fixed the comment: should be rebase on Github